tests/integration/test_s3_storage_conf_proxy/test.py

```diff
@@ -17,6 +17,22 @@ def run_resolver(cluster):
     )
     cluster.exec_in_container(container_id, ["python", "resolver.py"], detach=True)
 
+    for i in range(10):
+        response = cluster.exec_in_container(
+            container_id,
+            [
+                "curl",
+                "-s",
+                f"http://resolver:8080/hostname",
+            ],
+            nothrow=True,
+        )
+        if response == "proxy1" or response == "proxy2":
```

Comment:
Can proxy resolver produce something other than pre-configured values?

Reply:
Nope, it's hardcoded to work only with `proxy1` and `proxy2`. Check https://github.com/arthurpassos/ClickHouse/blob/master/docker/test/integration/runner/compose/docker_compose_minio.yml#L23

Reply:
then why do you need this `if` ?

Reply:
I guess it's better than asserting for an empty response and assuming it's correct?